crates/holochain/src/conductor/conductor.rs

```diff
@@ -3268,6 +3265,7 @@ mod accessor_impls {
             self.spaces.dht_db(dna_hash)
         }
 
+        #[allow(dead_code)]
```

Comment:
I've put some dead_code allows around for calls that will probably be needed again as we're resurrecting commented out tests/etc.